### PR TITLE
Add SE_CAR_V2 price calculator template

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

- Add Car price template and CMS page
- Fix Registration Number field not showing previous value when going back to edit form section

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4518?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4518** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4518?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4517** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4517?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4516** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4516?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4513** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4513?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4510** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4510?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4508** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4508?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4507** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4507?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4506** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4506?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alebedev and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/priceCalculator/FormGridNew.tsx [hunk 1]
```diff
@@ -43,3 +41,13 @@ export function FormGridNew({ items, autofocusFirst, submitLabel }: FormSectionP
...
+
+// GOTCHA: Uncontrolled fields keep very first defaultValue they had
+// Therefore, we want to remount the field when default changes from empty to non-empty
+// But not when default changes from one value to another - this risks loosing user input
+const fieldKey = (field: InputField): string => {
+  const hasNonEmptyValue =
+    (field.defaultValue != null && field.defaultValue !== '') ||
+    (field.value != null && field.value !== '')
+  return `${field.name}_${hasNonEmptyValue}`
+}
```
Comment (guilhermespopolin) [inline apps/store/src/features/priceCalculator/FormGridNew.tsx:53]: Nice cleanup

**Review by guilhermespopolin — COMMENTED** [review]
🙌🏻

**Review by guilhermespopolin — APPROVED** [review]

**Comment by alebedev** [comment]
### Merge activity

* **Jul 23, 10:18 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4518) rebased this pull request as part of a merge.

**Pull request merged**